roles/KubevirtNodeLabeller/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 # defaults file for KubevirtNodeLabeller
 wait: true
+use_kvm: true
```

Comment:
do we want to have this change in the node-labeller upstream/standalone project?
if not, we have a fork here for all intents and purposes (which could be perfectly fine as long we acknowledge this).

Reply:
node-labeller already contains this logic.

Reply:
Great then.